app/views/api/v1/apps/show.rabl

```diff
@@ -0,0 +1,3 @@
+object @application
+
+attributes :id, :name, :website
```

Comment:
Do we need to expose the id here? (I don't think so). With the requirement of apps self-registering on different instances, there's probably little value to tracking numeric IDs, since two users using the same app from their different phones with the same instance will have different app credentials in the db.

Btw if anyone has ideas on how to improve that authorization flow, I'm all ears. Had to invent it since GNU social doesn't even try to do OAuth properly, they just use password grants/basic auth, and I haven't heard of any other federated networks with OAuth APIs.